### DISCUSSION TITLE
Show duration change at history tab

### DISCUSSION THREAD

**Discussion by dremdem** [allure-framework/allure2#2250] [Ideas & Feedback]
Follow-up from #604 

Quote from the original issue: 

> Add an duration for TC run as well. It'll also would be nice to add some traction arrows to Duration entry(arrows on the screenshot are for illustration only)

![image](https://github.com/allure-framework/allure2/assets/17809415/eb094f74-9ae8-4687-8392-56185bcdb62d)

# Rationale 

We have group of tests that should run no longer then given threshold. 
So, our daily routine to check the slowest tests from this group and it will nice to have the history 
of the run duration of the certain test.

**Comment by baev** [comment]
Do you have any other ideas on what information/widgets/etc can help you with your problem?

**Reply by dremdem** [reply]
In general, we can start with the analysis of the `Duration trend` graph: 

<img width="1153" alt="image" src="https://github.com/allure-framework/allure2/assets/17809415/e42dad89-138b-4d8b-9430-5130fe3b0db3">

- if we discover some drastic rises in the duration, we need to know who the suspect is
- then we can go to the `Behaviours` tab, sort by duration and see the top N slowest
<img width="1163" alt="image" src="https://github.com/allure-framework/allure2/assets/17809415/6bab926c-c62e-4fe1-91d9-a61a3dfdb362">

- and final step is to understand what is the duration trend of every slow test. That's why we need this feature that will show the duration of the exact test
